src/native/managed/compile-native.proj

```diff
@@ -31,9 +31,6 @@
     <ItemGroup>
         <!-- additional native compilation-specific properties to pass down to the ProjectReference -->
         <SubprojectProps Include="NativeLib" Value="$(NativeLibKind)"/>
-
-        <SubprojectProps Condition="'$(SysRoot)' != ''" Include="SysRoot" Value="$(SysRoot)" />
-        <SubprojectProps Condition="'$(LinkerFlavor)' != ''" Include="LinkerFlavor" Value="$(LinkerFlavor)" />
         <SubprojectProps Condition="'$(CustomLinkerArgToolchainArg)' != ''" Include="CustomLinkerArgToolchainArg" Value="$(CustomLinkerArgToolchainArg)" />
```

Comment:
```suggestion
```
?

Reply:
Yes, and I'm figuring out which distro or OS will ever need this:
https://github.com/dotnet/runtime/blob/2cd3a53d7fb020fe67549e0a18ff97e5f950a9d9/src/native/managed/native-library.props#L22-L24
"no soname" is as good as having an "soname without version". On gnu distro with nativeaot published shared lib, we get:

```
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [ld-linux-aarch64.so.1]
 0x000000000000001d (RUNPATH)            Library runpath: [$ORIGIN]
```
with both bfd and lld, and on musl, we get

```
 0x0000000000000001 (NEEDED)             Shared library: [libc.musl-aarch64.so.1]
 0x000000000000000f (RPATH)              Library rpath: [$ORIGIN]
```
there is no hardcoded name to worry about.